cha-ching-app/src/SubscriptionButton.js

```diff
@@ -3,15 +3,12 @@ import PropTypes from "prop-types";
 
 const SubscriptionButton = ({ text, onClick = () => {}, selected = false }) => {
   return (
-    <div className="buttonWrapper">
       <button
         onClick={onClick}
-        className="SubscriptionButton"
-        style={{ backgroundColor: selected ? "#d7f8be" : "#d9d9d9" }}
+        className={`SubscriptionButton ${selected ? "highlightedButton" : ""}`}
```

Comment:
Nice! since we did this with CSS classes, we will be able to unit test that the class changes onClick